src/Hintman/Config.hs

```diff
@@ -0,0 +1,43 @@
+module Hintman.Config
+    ( HintmanConfig(..)
+    , defaultHintmanConfig
+    , loadFileConfig
+    ) where
+
+import Toml (AnyValue (..), BiMap (..), BiToml, (.=))
+
+import qualified Toml
+
+
+
+data Check = TrailingSpaces
+           | TrailingNewlines deriving (Eq, Read, Show)
+
+newtype HintmanConfig = HintmanConfig
+    { hintmanConfigChecks :: [Check]
+    } deriving (Show)
+
+defaultHintmanConfig :: HintmanConfig
+defaultHintmanConfig = HintmanConfig [TrailingSpaces, TrailingNewlines]
+
+showCheck :: Check -> Text
+showCheck TrailingSpaces   = "trailing-spaces"
```

Comment:
It's better to use `LambdaCase` extension to remove some code duplication. e.g.:

```haskell
showCheck = \case
    Trailing ->...
```